javascript-practice/src/javascripts/models/product.js

```diff
@@ -0,0 +1,13 @@
+import APIService from "../services/service"
+
+export default class ProductModel {
+	constructor() {
+		this.apiService = new APIService("/products")
+	}
+
+	/**
+	 * Method to retrieve the list of product from the API.
+	 * @returns {Promise<Object[]>} A promise that resolves with the list of product.
+	 */
+	getProductList = () => this.apiService.getList()
+}
```

Comment:
The model but it does carry any structure of Product. Pls help to update. @mthanh2209

Reply:
Thank you. I updated it